test/testfilelister.cpp

```diff
@@ -79,6 +80,11 @@ class TestFileLister: public TestFixture {
         // Make sure headers are not added..
         ASSERT(files.find("lib/tokenize.h") == files.end());
     }
+
+    void fileExists() const {
+        ASSERT_EQUALS(false, FileLister::fileExists("lib"));
+        ASSERT_EQUALS(true, FileLister::fileExists("readme.txt"));
```

Comment:
Please remove this assertion. The testrunner binary is placed in different paths.. we cant expect that there will be a readme.txt in the build output folder.

The previous assertion is fine. If testrunner is placed in cppcheck root path then it checks that `fileExists` returns false for a existing folder. Otherwise the assertion will check that `fileExists` returns false for a non-existing path.. that is good also. The problem would be if the build output folder has a file with the name "lib" but we can maybe ignore that problem.

Reply:
We check if we are in the correct place during the test setup see line 41 of testfilelister.cpp
I can still remove it but for me it seems like a valid test.